### ISSUE TITLE
DoubleCurlyIf false positive on multiline expressions

### ISSUE THREAD

**Issue by TWiStErRob** [TWiStErRob/net.twisterrob.ghlint#222]
### Version

ghlint 0.5.0

### Reproduction steps

Run `ghlint` on:
   ```yaml
   if: >
     ${{
       github.event_name == 'workflow_dispatch' || 
       github.event_name == 'workflow_call' ||
       (github.event_name == 'pull_request' && github.event.pull_request.draft == false && github.event.pull_request.merged == true)
     }}
   ```

### Expected behavior

Pass

### Actual behavior

> if: is not wrapped in double-curly-braces.

**Comment by TWiStErRob** [comment]
Testing https://github.com/TWiStErRob/github-actions-test/pull/27
